src/DropdownMenu.js

```diff
@@ -16,11 +70,19 @@ const DropdownMenu = React.createClass({
         'dropdown-menu-right': this.props.pullRight
       };
 
+    // Remove outline on focus by default
+    let style = this.props.style || {};
+    style.outline = style.outline || 'none';
```

Comment:
You need to be able to listen for the keyDown events for traversing though. Not sure how you can do that without focusing the list element itself? Maybe you have some global stuff to handle that?